floyd/manager/experiment_config.py

```diff
@@ -26,5 +26,25 @@ def get_config(cls):
             raise FloydException("Missing .floydexpt file, run floyd init first")
 
         with open(cls.CONFIG_FILE_PATH, "r") as config_file:
-            experiment_config_str = config_file.read()
-        return ExperimentConfig.from_dict(json.loads(experiment_config_str))
+            experiment_config = json.loads(config_file.read())
+
+        # Get the family_id from the API and write it to the file
+        if not experiment_config['family_id']:
+            return cls.set_experiement_config_from_name(experiment_config['name'])
+
+        return ExperimentConfig.from_dict(experiment_config)
+
+    @classmethod
+    def set_experiement_config_from_name(cls, name):
+        from floyd.client.project import ProjectClient
+        project = ProjectClient().get_by_name(name)
+
+        if not project:
+            raise FloydException('Project not yet initialized. Run `floyd init <project_name>` before continuing.')
+
+        e = ExperimentConfig(name, project.id)
```

Comment:
Nit: Can you switch to an expanded variable name? eg: `experiment_config`. Since we also have experiments, this variable name is ambiguous.